### ISSUE TITLE
blockreassurance: item by location

### ISSUE THREAD

**Issue by jf-viguier** [PrestaShop/PrestaShop#15932]
Thanks for the great 4.0 version of blockreassurance +1
For me, a feature is missing : choosing witch items to display for each location.
i.e. I want item 1 & 2 in the footer, 3 & 4 in product page, ...

**Comment by khouloudbelguith** [comment]
Hi @jf-viguier,

Thanks for your report.
>choosing witch items to display for each location

Choose the position of each item like this?
![image](https://user-images.githubusercontent.com/16067358/66754687-d3f40300-ee96-11e9-8c4e-1ffb8e6c845f.png)

Thanks!

**Comment by jf-viguier** [comment]
Yes @khouloudbelguith, like this.

In real life stores, you don't repeat items in footer, product page, cart page but write different texts.

**Comment by khouloudbelguith** [comment]
@jf-viguier, thanks for these clarifications.
Ping @PrestaShop/prestashop-product-team what do you think?

Thanks!

**Comment by colinegin** [comment]
Hello @jf-viguier,

thanks for your kind words and feedback ! 
I agree your idea would be a great feature improvement. 
We need to define it a bit more in terms of specs and design, and will add it in a future version.

Thanks again !

**Comment by colinegin** [comment]
Hello @TristanLDD ,

@Progi1984 is currently working on fixing some bugs on blockreassurance module.
Do you think you will have time to update design to add this feature or we'll keep it for a future version of the module ?

**Comment by TristanLDD** [comment]
@colinegin Yes I can do the mockups, I will post them here as soon as possible.

**Comment by TristanLDD** [comment]
@Coline, I had a closer look at the module and it appears that this feature will require some significant changes in the interface and code. 

Adding a column in the table as @khouloudbelguith suggested will actually not work because the module is made in a way that you can't choose the position of a specific item but the position of a group of several items (up to 8) in the "Display" tab. 

So we would have to: 
1 - Add the possibility to create several groups of items : 
![Capture d’écran 2019-11-08 à 16 15 44](https://user-images.githubusercontent.com/27484094/68487407-4f9d6180-0243-11ea-8059-9eedb2a647f2.png)


2 - Add the possibility to assign a specific position to a group of items in the "Display" tab:
![Capture d’écran 2019-11-08 à 16 14 05](https://user-images.githubusercontent.com/27484094/68487433-5c21ba00-0243-11ea-9d6b-1a495d9e963f.png)


3 - If we want to make every group of items customizable, we would also have to change the third tab "Appearance": 
![Capture d’écran 2019-11-08 à 16 13 58](https://user-images.githubusercontent.com/27484094/68487479-6e9bf380-0243-11ea-8b20-8e75dabb5f8e.png)
We could also choose to apply the same style to all groups of items and keep this tab as it is. 

As you can see, this feature is actually not super easy to implement.

**Comment by jf-viguier** [comment]
The link widget already have a block management : 
http://doc.prestashop.com/display/PS17/Link+Widget

**Comment by colinegin** [comment]
@TristanLDD thanks for the information, I would then recommend to leave this feature for a future version as this version is more dedicated to fixing bugs. Is it ok for you @Progi1984 ?

**Comment by Progi1984** [comment]
@colinegin As you want ;)

**Comment by Hlavtox** [comment]
A custom module can solve this in an infinite amount of ways. :-)
